src/libraries/Microsoft.Extensions.Configuration.Binder/tests/Common/ConfigurationBinderTests.Collections.cs

```diff
@@ -1862,5 +2331,9 @@ public void TestMutatingDictionaryValues()
             Assert.Equal("InitialValue", dict["Key"][0]);
             Assert.Equal("NewValue", dict["Key"][1]);
         }
+
+        // Test behavior for root level arrays.
+
+        // Tests for TypeConverter usage.
```

Comment:
are these remaining TODOs? If so I'd recommend removing and replacing with issue